tests/Rector/Namespace_/FactoryDefinitionRectorConfiguredNoMatch/Fixture/without_args.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace RectorLaravel\Tests\Rector\Namespace_\FactoryDefinitionRector\Fixture;
+
+use RectorLaravel\Tests\Rector\Namespace_\FactoryDefinitionRector\Source\Model;
+
+$factory->define(Model::class, function () {
+    // ...
+});
+
+$factory->state(Model::class, 'address', function () {
+    // ...
+});
+
+$factory->afterMaking(Model::class, function (Model $user) {
+    // ...
+});
+
+$factory->afterCreating(Model::class, function () {
+    // ...
+});
+
+?>
+-----
+<?php
+
+namespace RectorLaravel\Tests\Rector\Namespace_\FactoryDefinitionRector\Fixture;
+
+use RectorLaravel\Tests\Rector\Namespace_\FactoryDefinitionRector\Source\Model;
+
+$factory->define(Model::class, function () {
+    // ...
+});
+
+$factory->state(Model::class, 'address', function () {
+    // ...
+});
+
+$factory->afterMaking(Model::class, function (Model $user) {
+    // ...
+});
+
+$factory->afterCreating(Model::class, function () {
+    // ...
+});
+
+?>
```

Comment:
This can be removed as a fixture file can just contain the non changing code and it will act as a test to prove that nothing have been changed